src/components/layouts/MainLayout.jsx

```diff
@@ -1,8 +1,231 @@
+import { Suspense, useState } from 'react'
 import { Outlet } from 'react-router-dom'
+import styled, { css } from 'styled-components'
 
-export const MainLayout = () => (
-  <>
-    main data
-    <Outlet />
-  </>
-)
+import { useTranslation } from 'react-i18next'
+import { BurgerMenuIcon, MoonIcon, SunIcon } from '../../assets/icons'
+import { useAppThemeContext } from '../../styles/theme/provider'
+import { SideBar } from '../SideBar'
+import {
+  useBreakpointValue,
+  ScreenWrapper,
+  getBreakpointsStyles,
+  getDesktopStyles,
+} from '../../styles/breakpoints'
+import { PrimaryColorBtn } from '../PrimaryColorBtn'
+import { Text } from '../Text'
+
+export const MainLayout = () => {
+  const { t } = useTranslation()
+  const [isBurgerMenuOpen, setIsBurgerMenuOpen] = useState(false)
+  const [selectedTab, setSelectedTab] = useState('calendar')
```

Comment:
We will need to connect it to routes. So would be better to change, but for now let's keep it as is